types/function-bind/function-bind-tests.ts

```diff
@@ -1,5 +1,7 @@
 import bind = require('function-bind');
 
+declare function call<T, A extends any[], R>(this: (this: T, ...args: A) => R, thisArg: T, ...args: A): R;
```

Comment:
any reason to keep `any` here?
```suggestion
declare function call<T, A extends unknown[], R>(this: (this: T, ...args: A) => R, thisArg: T, ...args: A): R;
```

Reply:
For this PR, I'm just swapping ts4.0's contents up to the root as part of DT maintenance, so I don't want to make any other changes. This is a good idea for a follow-up PR, though.